qir/qsharp/NegativeBaseProfiles/ExternalCall/QSharpVersion/Example.qs

```diff
@@ -0,0 +1,16 @@
+namespace LoopCase {
```

Comment:
Calling this `LoopCase` is a bit confusing since it doesn't involve any loops.

Reply:
Updated to match the name in the test.